src/main/java/org/spongepowered/api/event/item/inventory/CraftItemEvent.java

```diff
@@ -56,7 +56,7 @@
      * This event is fired before the item is taken out of the
      * output slot but after completing the recipe in the grid.
```

Comment:
"This event is fired when a recipe has been completed and is rendering in the crafting output slot, but before the recipe is actually crafted."